### ISSUE TITLE
Comments on Figure 4

### ISSUE THREAD

**Issue by aphearin** [npadmana/biasVmax#4]
1. In the caption, simply state without explanation that the figure only shows Bolshoi results. The explanation will appear in the text. 
2. In general, this caption is too wordy. Since the figure is identical to previous ones, you don't need to re-explain the axes. The final few sentences can be replaced with something simpler like, "As can be seen by comparing these results to those in Figures 2 & 3, the Vmax-dependence of halo bias is dramatically reduced by excluding ejected subhalos." 
3. It may also be good to include a concluding caption sentence spelling out explicitly: "This implies an intimate connection between assembly bias and subhalo back-splashing." 
4. All my cosmetic comments from Figures 2 & 3 carry over directly here, though this figure doesn't need to be as large as Figure 3 does (the current size is good).

**Comment by npadmana** [comment]
I wonder if we should drop the linear bias figure here. The effect seems pretty marginal and it distracts from the much more impressive figure on the right.

Is there any residual scale dependence above 1 Mpc/h? Do we understand the drop-off below 1 Mpc/h?
